.changeset/hungry-nights-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@prairielearn/postgres': minor
```

Comment:
@reteps @miguelaenlle removal of functions is always a breaking change and thus always a major version. I'll open a PR to fix this before https://github.com/PrairieLearn/PrairieLearn/pull/12497 is merged.